### ISSUE TITLE
Chromedriver latest version not downloaded in Docker

### ISSUE THREAD

**Issue by BajajGirik** [BajajGirik/deal-finder#33]
## Overview:
If you look at our current `Dockerfile`, you will find the below line in it.

```Dockerfile
RUN wget -O /tmp/chromedriver.zip http://chromedriver.storage.googleapis.com/`curl -sS chromedriver.storage.googleapis.com/LATEST_RELEASE`/chromedriver_linux64.zip
```

This line tries to download chromedriver for `linux64` with a version that is retrieved by [chromedriver.storage.googleapis.com/LATEST_RELEASE](https://chromedriver.storage.googleapis.com/LATEST_RELEASE) api. This api has been deprecated and returns the latest version till `114` (i.e.`114.0.5735.90`) which is not the latest version now.

The latest stable version for chrome went up to `116.0.5845.96`. Now when we build the image using `Dockerfile` and run the container, it throws the following error.

```bash
Traceback (most recent call last):
  File "/deal_finder/main.py", line 1, in <module>
    from discord_client import DiscordWrapper
  File "/deal_finder/discord_client/__init__.py", line 1, in <module>
    from discord_client.client import DiscordWrapper
  File "/deal_finder/discord_client/client.py", line 6, in <module>
    from discord_client.product_tracker import ProductTrackerCog
  File "/deal_finder/discord_client/product_tracker.py", line 6, in <module>
    from utils.discord import DiscordUtils
  File "/deal_finder/utils/__init__.py", line 1, in <module>
    from utils.product_meta_collector import ProductMetaCollector
  File "/deal_finder/utils/product_meta_collector.py", line 11, in <module>
    web_browser = WebBrowser()
  File "/deal_finder/web_browser.py", line 20, in __init__
    self.__driver =  webdriver.Chrome(options=chrome_options)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 49, in __init__
    super().__init__(
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 54, in __init__
    super().__init__(
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 206, in __init__
    self.start_session(capabilities)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 291, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 346, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 245, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114
Current browser version is 116.0.5845.96 with binary path /usr/bin/google-chrome
Stacktrace:
#0 0x564b5f51d4e3 <unknown>
#1 0x564b5f24cc76 <unknown>
#2 0x564b5f27a04a <unknown>
#3 0x564b5f2754a1 <unknown>
#4 0x564b5f272029 <unknown>
#5 0x564b5f2b0ccc <unknown>
#6 0x564b5f2b047f <unknown>
#7 0x564b5f2a7de3 <unknown>
#8 0x564b5f27d2dd <unknown>
#9 0x564b5f27e34e <unknown>
#10 0x564b5f4dd3e4 <unknown>
#11 0x564b5f4e13d7 <unknown>
#12 0x564b5f4ebb20 <unknown>
#13 0x564b5f4e2023 <unknown>
#14 0x564b5f4b01aa <unknown>
#15 0x564b5f5066b8 <unknown>
#16 0x564b5f506847 <unknown>
#17 0x564b5f516243 <unknown>
#18 0x7f9c34bc5044 <unknown>
```

Basically the chromedriver is not upto date and doesn't support the latest version of chrome.
Our goal is to fix the flow used for downloading chromedriver in the `Dockerfile`.

## References for chromedriver changes:
1. https://chromedriver.chromium.org/downloads
2. [Latest endpoints that can be used for newer chromedriver versions](https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints)

## Side tasks:
1. Change Dockerfile first line from `FROM python:3.9.17` to `FROM --platform=linux/amd64 python:3.9.17` (Thanks to @ishita-chandra ).
